src/commands/restriction/account.ts

```diff
@@ -72,9 +81,20 @@ export default class extends ProfileCommand {
     execute(options: CommandOptions) {
         this.spinner.start();
         const profile = this.getProfile(options);
+
+        const password = options.password || readlineSync.question('Enter your wallet password: ');
```

Comment:
If the user provides an address, they should not be asked for the password (applies to other commands)